Gemfile.lock

```diff
@@ -164,14 +164,14 @@ GEM
     ast (2.4.2)
     awrence (1.2.1)
     aws-eventstream (1.3.0)
-    aws-partitions (1.792.0)
+    aws-partitions (1.943.0)
     aws-sdk-cloudwatchlogs (1.49.0)
       aws-sdk-core (~> 3, >= 3.122.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-core (3.179.0)
-      aws-eventstream (~> 1, >= 1.0.2)
+    aws-sdk-core (3.197.0)
```

Comment:
should we specify the required min versino in the Gemfile?

```ruby
gem 'aws-sdk-core', '>= 3.188.0'
```

Reply:
good idea, done